engine/src/main/java/com/datatorrent/stram/stream/OiOStream.java

```diff
@@ -29,11 +29,12 @@
  *
  * @since 0.3.5
  */
-public class OiOStream implements Stream, SweepableReservoir
```

Comment:
Let's say upstream operator puts n tuples into OiOStream, both `Sink.getCount(true)` and `SweepableReservoir.getCount(true)` should return n and reset count to 0. If there is only one count, only one call (whichever comes first) would return n, the second one would return 0.